Jenkinsfile

```diff
@@ -125,6 +125,11 @@ pipeline {
                         body: '${DEFAULT_CONTENT}'
                     }
                 }
+                if (!env.BRANCH_NAME.startsWith('PR-')) {
+                    configFileProvider([configFile(fileId: 'maven-settings-with-deploy-snapshot', variable: 'MAVEN_SETTINGS')]) {
+                        sh "$MAVEN_HOME/bin/mvn deploy -B -V -e -s $MAVEN_SETTINGS -Dmaven.test.failure.ignore=true -Dansi.strip -DskipTests=true"
```

Comment:
I suggest using `mvn deploy:deploy` so that we don't compile or run the tests again.